doc/usage.md

```diff
@@ -153,6 +153,9 @@ nil
        (repl/message {:op "eval" :code "(+ 2 3)"})
        repl/response-values))
 ```
+If your nREPL server is running on a different machine or listening on a specific
```

Comment:
Please add a blank line after the code block.

Reply:
Done in the updated commit.